Makefile

```diff
@@ -1,6 +1,7 @@
 DESCRIPTION = CS50 Library for Java
 MAINTAINER = CS50 <sysadmins@cs50.harvard.edu>
-NAME = lib50-java
+OLD_NAME = lib50-java
+NAME = libcs50-java
```

Comment:
Did we deploy this to CS50 IDE already? (I.e., indeed necessary to have `OLD_NAME`?)